src/System.Private.ServiceModel/src/System/ServiceModel/Description/CallbackDebugBehavior.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.ServiceModel.Channels;
+using System.ServiceModel.Dispatcher;
+
+namespace System.ServiceModel.Description
+{
+    public class CallbackDebugBehavior : IEndpointBehavior
+    {
+        bool includeExceptionDetailInFaults = false;
+
```

Comment:
This is missing accessibility modifier (private), and isn't using the correct naming convention. This field can go away though if you convert IncludeExceptionDetailsInFaults into an auto property.